config/splat.us.dra.yaml

```diff
@@ -118,7 +118,7 @@ segments:
       - [0x419B0, rodata] # jpt_80114388
       - [0x419D0, .rodata, 75DA0]
       - [0x419E8, .rodata, 75F54] # jpt_8011643C
-      - [0x41A00, rodata] # jpt_801169F4
+      - [0x41A00, .rodata, 75F54] # jpt_801169F4
```

Comment:
I went ahead and updated all the comments in this file, let me know if that is improper. Probably should be its own PR but since it's just comments hopefully it's okay?